app/controllers/maps_controller.rb

```diff
@@ -86,6 +86,20 @@ def usermaps
     end
   end
 
+  # GET maps/new
+  def new
+    @map = Map.new(:name => "Untitled Map", :permission => "public", :arranged => true)
```

Comment:
In new ruby you can rewrite to

    @map = Map.new(name: "Untitled Map", permission: "public", arranged: true)

Hold on!! Any reason you went with public and not commons?